cypress/e2e/awx/views/jobs.cy.ts

```diff
@@ -110,104 +109,6 @@ describe('Jobs: List', () => {
   });
 });
 
-describe('Jobs: Delete', () => {
```

Comment:
what's the reason to remove delete tests for jobs? i think we could revamp these tests instead of removing them

Reply:
I've removed them from this file and put them in `Jobs.cy.tsx`